internal/graph/schema/vulnerability.graphqls

```diff
@@ -476,13 +476,28 @@ input UpdateImageVulnerabilityInput {
 	suppress: Boolean!
 	"New state of the vulnerability."
 	state: ImageVulnerabilitySuppressionState
+	"The unique identifier of the vulnerability. E.g. CVE-****-****."
+	identifier: String!
```

Comment:
Trenger vi dette feltet som input, samme kanskje med de to under. Ligger ikke de i `vulnerabilityID`?